spyder/app/tests/test_mainwindow.py

```diff
@@ -166,6 +166,52 @@ def test_calltip(main_window, qtbot):
 
 
 @flaky(max_runs=3)
+@pytest.mark.skipif(os.name != 'nt' and not PY2 and PYQT5 or os.name == 'nt' and PY2,
```

Comment:
This test is also segfaulting on Python 2 and PyQt5 on Linux. So I think the right skipping condition should

    not (os.name != 'nt' and PYQT5)

or something like that.